### PR TITLE
created non-auth-landing-page

### PR DESCRIPTION
**Pull request by roxannebobby**
## Description

- changed function component to an arrow function
- deleted massive code in the css file, and changed the file to .less
- used ant.design components
- created login button incorporating the new Auth0 authorization link
- created Apply as Mentee and Apply as Mentor buttons
- created a Learn More link that points to the /apply page as per ticket
- reported new issue that occurs when landing on a non-authenticated page

This non-auth page was needed for those who landed within the app and did not yet have an account or had not yet been authenticated.

Fixes # (issue)

[Link here](https://bloomtechlabs.atlassian.net/browse/BL-997?atlOrigin=eyJpIjoiYWZmYzRkMzQ4M2YzNDU1NWFiZmU1MzY2NGNkNDI0NTEiLCJwIjoiaiJ9)

## Loom Video

[Link here](https://www.loom.com/share/922c64c1ff14401bb9e8f4f3c10b1665)

## Type of change

- [X] New feature (breaking change (issue has been reported) which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by nikolayglushetskiy — APPROVED** [review]
Looks goos!

**Review by zoebrantley — APPROVED** [review]
Looking great!!!

**Review by TylrPopcorn — DISMISSED** [review]
I like these changes :)

File: src/components/pages/LandingPage/LandingPage.less [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+/* content */
+.content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  margin-top: 30px;
+}
+
```
Comment (jzkime) [inline src/components/pages/LandingPage/LandingPage.less:40]: I like that they are broken up in sections. 

There seem to be spaces in between these styles, but no spaces in Section 1 - a format clean up might be nice to keep it consistent.

**Review by jzkime — APPROVED** [review]
It looks good, the LandingPage component looks clean, and straight forward; great for future developers of this team. Good work.

**Review by JamesFincher — DISMISSED** [review]
Great work!

**Review by ashurik522 — DISMISSED** [review]
Looks good, I look forward to seeing it merged and testing it out!

**Pull request merged**